### DISCUSSION TITLE
May I ask how to use this plugin using the CupertinoPageScaffold style

### DISCUSSION THREAD

**Discussion by gracewang9** [fluttercandies/flutter_wechat_assets_picker#467] [Q&A]
My style is using IOS style Cupertino, while using wechat_ Assets_ Picker: encountered a problem during ^ 8.6.1 and prompted 'No MaterialLocalizations found.', I have added Material, but Localization still causes the same error. May I know what I need to do to use this plugin correctly

**Comment by AlexV525** [comment]
Add `GlobalMaterialLocalizationsDelegate` to your `CupertinoApp` or whatever `WidgetsApp` you're using.
https://github.com/fluttercandies/flutter_wechat_assets_picker/blob/9caea36129208c6fbc6cafc6e86a2e8febc2413b/example/lib/main.dart#L51
https://github.com/fluttercandies/flutter_wechat_assets_picker/blob/9caea36129208c6fbc6cafc6e86a2e8febc2413b/example/lib/l10n/gen/app_localizations.dart#L83-L88
